DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
> ...couldn't we just reshape layer_3 and use `bidirectional_dynamic_rnn`

I'd really like to try to change one thing at a time and not many things at a time; thus stick with `bidirectional_rnn()` until issue 62 is solved.